ui/shared/components/panel/family.jsx

```diff
@@ -13,15 +13,20 @@ import TextFieldView from './view-fields/TextFieldView'
 import Sample from './sample'
 import { ColoredIcon, InlineHeader } from '../StyledComponents'
 import { VerticalSpacer, HorizontalSpacer } from '../Spacers'
+import { AddableSelect } from '../form/Inputs'
+import DataLoader from '../DataLoader'
 import {
   FAMILY_ANALYSIS_STATUS_OPTIONS,
   FAMILY_FIELD_ANALYSIS_STATUS,
+  FAMILY_FIELD_ASSIGNED_ANALYST,
   FAMILY_FIELD_ANALYSED_BY,
   FAMILY_FIELD_FIRST_SAMPLE,
   FAMILY_FIELD_RENDER_LOOKUP,
   FAMILY_FIELD_OMIM_NUMBER,
   FAMILY_FIELD_PMIDS,
 } from '../../utils/constants'
+import { getUserOptions, getUserOptionsIsLoading, getUsersByUsername } from '../../../pages/Project/selectors'
```

Comment:
One important thing to remember to do after you resolve merge conflicts is to run your code again and make sure you don't see any errors in the output for either the ui or the server. I'm pretty sure some of these functions got moved :)